src/pages/index.tsx

```diff
@@ -50,22 +54,33 @@ const Home: NextPage = () => {
   };
 
   useEffect(() => {
+    const controller = new AbortController();
+    const signal = controller.signal;
+
     // Initial fetch
-    const fetchAllData = async () => {
-      await fetchData("./status.json", setStatusData);
-      await fetchData("./incidents.json", setIncidentData);
-      await fetchData("./planned_maintenance.json", setPlannedMaintenanceData);
+    const fetchAllData = async (signal: AbortSignal) => {
+      await fetchData("./status.json", signal, setStatusData);
+      await fetchData("./incidents.json", signal, setIncidentData);
+      await fetchData(
+        "./planned_maintenance.json",
+        signal,
+        setPlannedMaintenanceData
+      );
     };
 
-    fetchAllData();
+    fetchAllData(signal).then();
 
     // Poll every 60 seconds
     const intervalId = setInterval(async () => {
-      await fetchData("./status.json", setStatusData);
+      await fetchData("./status.json", signal, setStatusData);
```

Comment:
Again not related to this PR, but the interval function doesn't need to be async or use await

Reply:
Same as above

Reply:
Doesn't need async

Reply:
Oops. Missed. Updated now.